src/layers/conv.jl

```diff
@@ -69,6 +76,10 @@ Keywords to control initialization of the layer:
 * `bias` - The initial bias vector is all zero by default. Trainable bias can be disabled entirely
   by setting this to `false`, or another vector can be provided such as `bias = randn(Float32, out)`.
 
+Convolutional layer can also be constructed with given weights and biases.
+While constructung a convolutional layer by providing weights and biases, the layer accepts the
+same keywords (and has the same defaults) as the `Conv((4,4), 3 => 7, relu)` method.
```

Comment:
```suggestion
Convolutional layers can also be constructed with given weights and biases. This constructor accepts the
same keywords (and has the same defaults) as the `Conv((4,4), 3 => 7, relu)` method.
```
"given" feels a tad non-descriptive, but I can't think of a concise way to describe this. "manual construction"? "directly constructed"?